Improve TLS docs (#1757)
pingcap/docs-tidb-operator: zh/enable-tls-for-mysql-client.md

    >
    > 创建的 Secret 对象必须符合上述命名规范，否则将导致 TiDB 集群部署失败。

2. 部署集群，设置 `.spec.tidb.tlsClient.enabled` 属性为 `true`：

    * 如果希望关闭 TiDB 服务端对客户端 CA 证书的认证，即在[配置 TiDB 服务端启用安全连接](https://docs.pingcap.com/tidb/stable/enable-tls-between-clients-and-servers#configure-tidb-server-to-use-secure-connections) 中不设置 ssl-ca 参数，可以设置 `.spec.tidb.tlsClient.disableClientAuthn` 属性为 `true`。
    * 如果希望跳过作为 MySQL 客户端的内部组件(如 TidbInitializer, Dashboard, Backup, Restore)的 CA 证书认证，可以设置 `.spec.tidb.tlsClient.skipInternalClientCA` 属性为 `true`。

>     * 如果希望跳过作为 MySQL 客户端的内部组件的 TLS 认证，可以给集群对应的 `TidbCluster` 加上 `tidb.tidb.pingcap.com/skip-tls-when-connect-tidb=“true”` 的 annotation。

Reviewer: ```suggestion
    * 如果希望跳过作为 MySQL 客户端的内部组件的 TLS 认证，可以给集群对应的 `TidbCluster` 加上 `tidb.tidb.pingcap.com/skip-tls-when-connect-tidb="true"` 的 annotation。
```

>     * 如果希望跳过作为 MySQL 客户端的内部组件的 TLS 认证，可以给集群对应的 `TidbCluster` 加上 `tidb.tidb.pingcap.com/skip-tls-when-connect-tidb="true"` 的 annotation。

Reviewer 2: 加一下从哪个版本开始支持

    > **注意：**
    >

>     > 为已部署的集群设置 `.spec.tidb.tlsClient.enabled` 属性为 `true`，将导致 TiDB Pod 重启。

Reviewer 2: ```suggestion
    > 已部署的集群 `.spec.tidb.tlsClient.enabled` 属性从 `false` 改为 `true`，将导致 TiDB Pod 滚动重启。
```

>     * 对于 TiDB Operator v1.1.0 及以后的版本，你可以给集群对应的 `TidbCluster` 加上 `tidb.tidb.pingcap.com/skip-tls-when-connect-tidb="true"` 的 annotation，来跳过作为 MySQL 客户端的内部组件(如 TidbInitializer, Dashboard, Backup, Restore)的 TLS 认证。
>     * 对于 TiDB Operator v1.3.0 及以后的版本，你可以设置 `.spec.tidb.tlsClient.disableClientAuthn` 属性为 `true`，来关闭 TiDB 服务端对客户端 CA 证书的认证，即在[配置 TiDB 服务端启用安全连接](https://docs.pingcap.com/tidb/stable/enable-tls-between-clients-and-servers#configure-tidb-server-to-use-secure-connections) 中不设置 ssl-ca 参数。
>     * 对于 TiDB Operator v1.3.0 及以后的版本，你可以设置 `.spec.tidb.tlsClient.skipInternalClientCA` 属性为 `true`，来跳过作为 MySQL 客户端的内部组件(如 TidbInitializer, Dashboard, Backup, Restore)的 CA 证书认证。

Reviewer 3: ```suggestion
    * 如需跳过作为 MySQL 客户端的内部组件（如 TidbInitializer、Dashboard、Backup、Restore）的 TLS 认证，你可以给集群对应的 `TidbCluster` 加上 `tidb.tidb.pingcap.com/skip-tls-when-connect-tidb="true"` 的 annotation。
    * 如需关闭 TiDB 服务端对客户端 CA 证书的认证，你可以设置 `.spec.tidb.tlsClient.disableClientAuthn` 属性为 `true`，即在[配置 TiDB 服务端启用安全连接](https://docs.pingcap.com/tidb/stable/enable-tls-between-clients-and-servers#configure-tidb-server-to-use-secure-connections) 中不设置 ssl-ca 参数。
    * 如需跳过作为 MySQL 客户端的内部组件（如 TidbInitializer、Dashboard、Backup、Restore）的 CA 证书认证，你可以设置 `.spec.tidb.tlsClient.skipInternalClientCA` 属性为 `true`。
```
这个 PR 只 cp 到 1.3 的话，可以不用说明版本了
